README.md

```diff
@@ -8,8 +24,36 @@ $ npm install
 $ npm start
 ```
 
-Currently the site is compiled to static HTML, CSS, and JS where it can run on github pages.
+## Develop Workflow
+
+After successfully building and running the server, go to localhost:8080/
+
+Now start changing files in /src and refresh localhost:8080/ You don't need to rerun the server, although it doesn't hurt to either.
```

Comment:
better to be clear in the dev flow: is a server reload not needed? If not, it's a bad idea to say that it's a good to do it anyway. What kind of edits necessitate reloading the server vs just reloading the browser? Documenting those would be quite valuable.